com.unity.cinemachine/Runtime/Core/InputAxisControllerBase.cs

```diff
@@ -238,13 +223,25 @@ static int GetControllerIndex(List<Controller> list, IInputAxisOwner owner, stri
             in IInputAxisOwner.AxisDescriptor axis, Controller controller)
         { 
         }
+        
+        /// <summary>
+        /// Called when the component is required to be reset.
+        /// </summary>
+        protected virtual void OnResetComponent() {}
+        
+        /// <summary>
+        /// Called when the inputs are updated
+        /// </summary>
+        protected virtual void OnUpdateInputs() {}
 
         void OnResetInput()
         {
             for (int i = 0; i < Controllers.Count; ++i)
                 Controllers[i].Driver.Reset(ref m_Axes[i].DrivenAxis());
         }
-
+        
+        //TODO Support fixed update as well. Input system has a setting to update inputs only during fixed update.
+        //TODO This won't work accuratly if this setting is enabled.
```

Comment:
True.

One option: don't implement Update() in the base class, leave it to the implementations to decide when they want to update.

I moved the code here to a new protected UpdateControllers() method that implementations can call from their Update or FicedUpdate.